scripts/release/version-updater.js

```diff
@@ -0,0 +1,195 @@
+#!/usr/bin/env node
```

Comment:
version update can be made with a one-line change via `sed` command. I don't want to review or maintain this 200-lines long script for a trivial change